prog/dns_zone/dns_zone_nexus.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "forwardable"
+
+require_relative "../../lib/util"
+
+class Prog::DnsZone::DnsZoneNexus < Prog::Base
```

Comment:
This prog will evolve into full fledged nexus that creates dns zone and dns server entities. It will have its own assemble and destroy methods etc., but this is what we need for initial version.